google-ads/src/main/resources/googleadsapi-primer/grpc_class_names

```diff
@@ -1,155 +1,363 @@
-io.grpc.Attributes$Builder
+io.grpc.alts.ComputeEngineChannelBuilder
 io.grpc.Attributes
+io.grpc.Attributes$Builder
+io.grpc.Attributes$Key
+io.grpc.auth.GoogleAuthLibraryCallCredentials
 io.grpc.auth.GoogleAuthLibraryCallCredentials$1
```

Comment:
Need to remove the nested classes - Class.forName() doesn't load these so it slows everything down.

Reply:
Actually maybe I'm misremembering... I can see that we originally had some in here.